content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -6,46 +6,163 @@ type: multi-code-lang
 code_lang_weight: 50
 ---
 
+## ASM capabilities 
+
+The following ASM capabilities are supported in the Node.js library, for the specified tracer version:
+
+| ASM capability                   | Minimum NodeJS tracer version |
+| -------------------------------- | ----------------------------|
+| Threat Detection  | 3.13.1|
+| Threat Protection  | 3.19.0  |
+| Open Source Vulnerability detection |  2.23.0 for NodeJS 12+, or 3.10.0 for NodeJS 14+ |
+| Custom Code Vulnerability detection  | 2.32.0 for NodeJS 12+, or 3.19.1 for NodeJS 14+ |
```

Comment:
_"Custom Code Vulnerability detection"_ is in beta, and it is in continuous improvement and we are adding more features whole the time.
For example, in the version that is in release process now, we are adding 3 new vulnerabilities detection and improved injection detection with new vulnerability sources.

Reply:
updated this section, will check with Gorka on suggested wording